content/applications/inventory_and_mrp/inventory/management/products/usage.rst

```diff
@@ -210,6 +217,36 @@ and create another one in a fresh pop-up window.
       :alt: Show packaging and contained quantities, specified on the product page form in the
             Inventory tab.
 
+Partial reservation
+-------------------
+
+After :ref:`completing the packaging setup <inventory/management/packaging-setup>`, packagings can
+be reserved in full or partial quantities for outgoing shipments. Partial packaging flexibility
+expedites order fulfillment by allowing immediate shipment of available items while awaiting the
+rest.
+
+To configure packaging reservation methods, go to :menuselection:`Inventory app --> Configuration
+--> Product Categories`. Then, click :guilabel:`Create` or select the desired product category. On
+the product category's form, in the :guilabel:`Logistics` section, :guilabel:`Reserve Packagings`
+can be set to :guilabel:`Reserve Only Full Packagings` or :guilabel:`Reserve Partial Packagings`.
```

Comment:
![Screenshot 2024-05-30 062337](https://github.com/odoo/documentation/assets/73958186/f543db7c-9efe-4e74-ba59-bcb98632c223)
On the 15.0 runbot I'm using for the review, I am not seeing a 'Reserve Packagings' field (in the Logistics section or anywhere else) on the Product Category form.
Is this a "runbot-related" issue? Does something else need to be enabled/configured before this field appears on a form? Either way, I think it might be worth looking into during your revisions, just to make sure everything is accurate. Thanks! 👍

Reply:
Thanks for pointing this out! I'll add an "important" admonition block so the reader will remember to enable the relevant feture.

Reply:
```suggestion
the product category's form, in the :guilabel:`Logistics` section, :guilabel:`Reserve Packagings`
can be set to :guilabel:`Reserve Only Full Packagings` or :guilabel:`Reserve Partial Packagings`.

.. important::

Ensure the TODO: product packagings feature is enabled
```